clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -103,14 +108,14 @@ do_some_syscalls()
     struct sigaction act;
     int res =
         syscall(SYS_rt_sigaction, /*too large*/ 1280, &act, nullptr, KERNEL_SIGSET_SIZE);
-    assert(res != 0 && errno == EINVAL);
+    CHECK(res != 0 && errno == EINVAL);
```

Comment:
Grepping the other burst*.cpp files shows they are all using assert.  Given (1) this file was already enabling assert in release build and (2) all the other files use assert, I'm not sure about this change.